src/main/java/org/simdjson/TapeBuilder.java

```diff
@@ -206,7 +206,13 @@ void reset() {
     }
 
     JsonValue createJsonValue(byte[] buffer) {
-        return new JsonValue(tape, 1, stringBuffer, buffer);
+        Tape newTape = new Tape(tape);
+
+        int stringBufferLen = stringParser.getStringBufferIdx();
+        byte[] newStringBuffer = new byte[stringBufferLen];
+        System.arraycopy(stringBuffer, 0, newStringBuffer, 0, stringBufferLen);
```

Comment:
Is this change related to https://github.com/simdjson/simdjson-java/issues/36? I'm asking because I'm a bit concerned that we need another allocation on the parsing path.

Reply:
And as I mentioned above, the default size of buffer and stringBuffer, as well as long[] tape in class Tape, is 34M. If we allocated 34M * 3 for each element, the cost is way to much.